src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -10,15 +10,40 @@ namespace Dynamo.Logging
     /// </summary>
     class AnalyticsService
     {
-        private static IAnalyticsUI adpAnalyticsUI = new ADPAnalyticsUI();
+        private static ADPAnalyticsUI adpAnalyticsUI = new ADPAnalyticsUI();
 
         /// <summary>
         /// Starts the client when DynamoModel is created. This method initializes
         /// the Analytics service and application life cycle start is tracked.
         /// </summary>
         /// <param name="model">DynamoModel</param>
-        internal static void Start(DynamoModel model)
+        /// <param name="disableADP">Pass true to disable ADP for the lifetime of the Dynamo or host process</param>
```

Comment:
sorry to pester on this...but I just saw you added a new serializable field called `disableADPForProcess`
I think the param here should be consistent with that. So either disableADPForProcess or disableADPin both places...
Asides from that...PR looks good

Reply:
@pinzart90 - what makes this field serializable? 
Also to be clear I added a private field to dynamoModel - and a public property to the startup configuration (which it not serialized)

I don't think either of them is serialized anywhere - but I can make them all match if you think the shorter, less specific name is better, and hopefully clients read the docs.

Reply:
Oh. It's private ..ok then.
I think it is better if they match...easier on debugging...
Since everything is internal...naming does not matter that much...
Any way you wish. I do not want to slow this PR down ...